torch/csrc/jit/passes/quantization.cpp

```diff
@@ -128,6 +128,39 @@ bool isFunctionNode(Node* n,
   return is_quantizable;
 }
 
+// If the op doesn't require observation, return
+// the the list of input indexes that we should check to see
+// if they are observed/quantized, if so, we can say the output
+// of this op is observed/quantized as well, since for these ops we can derive
+// the quantization parameters for output given inputs
+std::vector<size_t> getGeneralOpTensorInputIndexes(Node* n) {
+  std::vector<std::string> single_input_aten_funcs = {
```

Comment:
There is a bigger set of ops here: We should add:
```
max() — Returns the maximum value of the tensor (reduction only)
mean() — Mean function. Supported variants: reduction, dim, out
min() 
sort()
relu() 
avgpool2d() 
interpolate() - Interpolation
upsample() - Upsampling
upsample_bilinear() - Bilinear Upsampling
upsample_nearest()
```
https://pytorch.org/docs/stable/quantization.html#quantized-torch-tensor-operations

Reply:
actually right now relu is not in the list because we need to observe after conv-relu pair, maybe I need to change this logic to integrate that case, but for now I'll leave out relu in the list.